Cannot find module ./methods

Not sure what's up but it looks like `methods.js` isn't in the NPM package. 🤔

```
Error: Cannot find module './methods'
Require stack:
- /home/mauve/programming/dat-store/node_modules/@geut/hypercore-promise/index.js
- /home/mauve/programming/dat-store/node_modules/dat-sdk/index.js
- /home/mauve/programming/dat-store/test.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:980:15)
    at Function.Module._load (internal/modules/cjs/loader.js:862:27)
    at Module.require (internal/modules/cjs/loader.js:1042:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (/home/mauve/programming/dat-store/node_modules/@geut/hypercore-promise/index.js:1:48)
    at Module._compile (internal/modules/cjs/loader.js:1156:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1176:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Module.require (internal/modules/cjs/loader.js:1042:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/mauve/programming/dat-store/node_modules/@geut/hypercore-promise/index.js',
    '/home/mauve/programming/dat-store/node_modules/dat-sdk/index.js',
    '/home/mauve/programming/dat-store/test.js'
  ]
}
```

--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
     "lib",
     "bin",
     "index.js",
-    "callback-methods.js"
+    "methods.js"
   ],
   "scripts": {
     "start": "node index.js",

